internal/jasmine_node_test/jasmine_node_test.bzl

```diff
@@ -44,11 +32,71 @@ def jasmine_node_test(name, srcs = [], data = [], deps = [], **kwargs):
   all_data += [":%s_devmode_srcs.MF" % name]
   entry_point = "build_bazel_rules_nodejs/internal/jasmine_node_test/jasmine_runner.js"
 
-  nodejs_test(
+  nodejs_rule(
       name = name,
       data = all_data,
       entry_point = entry_point,
       templated_args = ["$(location :%s_devmode_srcs.MF)" % name],
       testonly = 1,
       **kwargs
   )
+
+def jasmine_node_test(name, srcs = [], data = [], deps = [], **kwargs):
+  """Runs tests in NodeJS using the Jasmine test runner and expects it to fail.
+
+  To debug the test, see debugging notes in `nodejs_test`.
+
+  Args:
+    name: name of the resulting label
+    srcs: JavaScript source files containing Jasmine specs
+    data: Runtime dependencies which will be loaded while the test executes
+    deps: Other targets which produce JavaScript, such as ts_library
+    **kwargs: remaining arguments are passed to the test rule
+  """
+  _jasmine_node_test(name = name, srcs = srcs, data = data, deps = deps, nodejs_rule = nodejs_test, **kwargs)
+
+def _xjasmine_node_test(
+  name,
+  size = None,
+  timeout = None,
```

Comment:
make this minimal, don't need all these attributes

Reply:
These are here so that they can be passed to native.sh_test while `**kwargs` is passed to `_jasmine_node_test`.

Reply:
the new rule can be short and undocumented since it's not in the public API
then you don't need this refactoring of the current rule, and don't need to pass through all the attrs